docs-source/content/userguide/introduction/introduction.md

```diff
@@ -48,6 +50,22 @@ In accordance with these policies, the operator and WebLogic Server are supporte
 Infrastructure using *Oracle Container Engine for Kubernetes*, or in a cluster running *Oracle Linux
 Container Services for use with Kubernetes* on OCI Compute, and on "authorized cloud environments".
 
+### Microsoft Azure Kubernetes Service
+
+[Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/) is a hosted Kubernetes environment.  The WebLogic Kubernetes
```

Comment:
WebLogic Kubernetes operator -> WebLogic Kubernetes Operator (when you use the full name, capitalize operator)

Reply:
got it :)